pkgs/elementsd/default.nix

```diff
@@ -1,33 +1,47 @@
-{ stdenv, fetchurl, pkgconfig, autoreconfHook, openssl, db48, boost, zeromq
-, zlib, qtbase ? null, qttools ? null, utillinux, protobuf, python3, qrencode, libevent
-}:
+{ stdenv, fetchurl, pkgconfig, autoreconfHook, openssl, db48, boost, zeromq, rapidcheck
+, zlib, miniupnpc, qtbase ? null, qttools ? null, wrapQtAppsHook ? null, utillinux, protobuf, python3, qrencode, libevent
```

Comment:
you shouldn't need `? null` on qtbase,qttools, etc.

I'm getting an evaulation error on these when withGui is true

Reply:
The function args just copied from the bitcoin nixpkg. The way this works is that instead of `callPackage` they use `libsForQt5.callPackage`. However, I'm getting a weird error after a bit of buidling

```
  GEN      qt/paymentrequest.pb.h
/nix/store/x1khw8x0465xhkv6w31af75syyyxc65j-gnused-4.7/bin/sed -e 's:\(res\/\(icons\|movies\)/\):/tmp/nix-build-elements-0.18.1.1.drv-0/elements-elements-0.18.1.1/src/qt/\1:' <"qt/bitcoin.qrc" >"qt/qrc_bitcoin_build.qrc"
make[2]: *** No rule to make target 'qt/res/rendered_icons/about.png', needed by 'qt/qrc_bitcoin.cpp'.  Stop.
```

For now, I removed the withGui = true commit again.